src/components/fieldHelpers/withValidationError.tsx

```diff
@@ -9,7 +9,7 @@ const WithValidationError: FC<Props> = ({ error, children }) => {
   return (
     <div className="relative pb-px">
       {error && (
-        <div className="text-xs text-salmon absolute bottom-validationError validation-errorMessage whitespace-no-wrap">
+        <div className="text-xs text-salmon absolute bottom-validationError validation-errorMessage whitespace-nowrap">
```

Comment:
oh noes... I thought tailwind wanted to be backwards-compatible most of the time 🙈 
so this update will be a breaking change for our apps :(